example/config/deploy.rb

```diff
@@ -4,3 +4,4 @@
 set :application, 'my_app_name'
 set :repo_url, 'git@github.com:MindscapeHQ/capistrano-raygun-deployment.git'
 set :deploy_to, '/var/www/my_app_name'
+set :raygun_deployment_environments, [:production]
```

Comment:
I think it looks good, but I think if you're requiring `:raygun_api_key` would be a option just detect if this key is applied and which env is being deployed.